docs/en/ingest-management/fleet/fleet-server.asciidoc

```diff
@@ -48,12 +48,11 @@ a self-managed cluster).
 bugfix releases)
 ** {kib} should be on the same minor version as {es}.
 
-//TODO: Uncomment this section when ECE 2.10 is available
-//* {ece} 2.10 or later
-//** Requires additional wildcard domains and certificates (which normally only
-//cover *.cname, not ..cname). This enables us to provide the URL for
-//{fleet-server} of `https://.fleet.`
-//** The deployment template must contain a slider for APM & Fleet.
+* {ece} 2.10 or later
+** Requires additional wildcard domains and certificates (which normally only
+cover *.cname, not ..cname). This enables us to provide the URL for
+{fleet-server} of `https://.fleet.`
+** The deployment template must contain a slider for APM & Fleet.
```

Comment:
I think the term "slider" is deprecated because its a dropdown now. Can you check on the new term?